### ISSUE TITLE
[roctracer] Build error, missing include files

### ISSUE THREAD

**Issue by QMechanic** [rocm-arch/rocm-arch#565]
When trying to install `roctracer-4.0.0-1` (which is required by `rocm-dev` which in turn is required by many other packages) it fails early due to missing include files like `kfd_ostream_ops.h` and `hip_ostream_ops.h`

Full logs can be seen at https://gist.github.com/QMechanic/73d5e5dffedaf49f481f34f82dad6edb

I have never had this before on the older versions of ROCm. Not sure if anyone else has this problem too.

**Comment by QMechanic** [comment]
Update. I just saw the `ModuleNotFoundError: No module named 'CppHeaderParser'` in the logs and realized that the python script that generates those header files was not working. So it turned out that I had `python-cppheaderparser` installed but the reason it wasn't working was because that was for python3.8 but now Arch uses 3.9, not sure why I didn't get this issue before though.

Anyways I solved this by running `pip3.9 install cppheaderparser`
